src/components/common/Loading.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import Spinner from './icons/Spinner';
+import { breakpoint } from '../../styles/globalStyles';
+import { Header, Container } from '../common/Layout';
+import { Title, Paragraph } from '../common/Typography';
+
+const Spacer = styled.div`
+  padding-top: 6rem;
+`;
+
+let title1 = 'Processing....';
```

Comment:
As we probably don't want to overwrite these strings - I see you have two sets of them - presumably to display when we have different loading states - they can be declared with `const`

Reply:
Done, thanks.

Reply:
@susanX @mr-bagglesworth Have I missed something? why are varibles declared and not props?

Reply:
@wright1 sorry for my late response - please see my comment below. If we can get / want to show various loading states, this would be useful.